tests/botan.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python
+
+"""This module contains a object that represents Tests for Botan analytics integration"""
+
+import os
+import unittest
+import sys
+sys.path.append('.')
+
+from telegram.utils.botan import Botan
+from tests.base import BaseTest
+
+class MessageMock(object):
+    chat_id = None
+
+    def __init__(self, chat_id):
+        self.chat_id = chat_id
+
+class BotanTest(BaseTest, unittest.TestCase):
+    """This object represents Tests for Botan analytics integration."""
+    token = os.environ.get('TOKEN')
```

Comment:
This is the bot token from BotFather. Please use `BOTAN_TOKEN` to get the token for the botan.io API (we'll set it up)